src/runner/index.js

```diff
@@ -81,6 +85,7 @@ export default class Runner extends EventEmitter {
             await Promise.race(promises);
         }
         catch (err) {
+            await browserSet.reportTaskStatus(TASK_STATUS.errored, err);
```

Comment:
Here we report error for all browsers, however as far as I understand it makes sense to report errors only in this particular browser.